frontend/src/lib/AlertAppError.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+    import { onMount } from "svelte";
+
+    let alertElement = document.querySelector("sl-alert.alert-app-error");
+
+    let errorMessage = "";
+
+    export let open = false;
+    export let closable = true;
+    export let variant = "danger";
+    export let duration = Infinity;
+    export let iconName = "exclamation-octagon";
+    export let ready = false;
+
+    export function show(error) {
+        errorMessage = error;
+        alertElement.show();
+    }
+
+    export function toast(error) {
```

Comment:
This function doesn't seem to be used.

Reply:
This component is wrapping `<sl-alert>` for generic usage.